### PR TITLE
Update the version number for amp-story to 1.0

### PR DESCRIPTION
**Pull request by newmuis**
This is a first step towards fixing #14357.  We move amp-story from 0.1 => 1.0, aliasing 0.1 as 1.0.  As a separate PR, we will copy 1.0 back to 0.1 to freeze it; this extra step is necessary to retain history properly.

### CODE REVIEW

**Comment by honeybadgerdontcare** [comment]
@newmuis You mention aliasing 0.1 as 1.0 but then mention copying 1.0 back to 0.1. Will it not remain an alias at that point (e.g. both 0.1 and 1.0 serve the same code)?

File: extensions/amp-story/1.0/test/test-amp-story.js [hunk 1]
```diff
@@ -68,6 +68,12 @@ describes.realWin('amp-story', {
   beforeEach(() => {
     win = env.win;
     element = win.document.createElement('amp-story');
+    // element.getAmpDoc = () => env.ampdoc;
```
Comment (newmuis) [inline extensions/amp-story/1.0/test/test-amp-story.js]: @choumx @erwinmombay mocking this all seems to work, but seems odd to have to do

**Comment by newmuis** [comment]
@honeybadgerdontcare At that point in time, yes, but we will continue development on 1.0 and not 0.1, so they will diverge over time.

**Comment by gmajoulet** [comment]
The tests are failing because they keep loading the ``0.1`` version of the ``amp-story`` extension.

As per https://github.com/ampproject/amphtml/blob/master/testing/describes.js#L692, unless you provide a specific extension version, it always use ``0.1``.

To fix the tests, we need to change ``extensions: ['amp-story']`` to ``extensions: ['amp-story:1.0']`` in ``test-amp-story.js``.

````
describes.realWin('amp-story', {
  amp: {
    runtimeOn: true,
    extensions: ['amp-story'],
  },
}, env => {});
````

with

````
describes.realWin('amp-story', {
  amp: {
    runtimeOn: true,
    extensions: ['amp-story:1.0'],
  },
}, env => {});
````

**Comment by newmuis** [comment]
Thanks @gmajoulet for tracking this down!

**Comment by newmuis** [comment]
@honeybadgerdontcare can you verify the validator changes?

**Comment by honeybadgerdontcare** [comment]
@newmuis github is saying theres conflicts for the test validator files, can you fix?

the protoascii change looks fine. if/when 1.0 starts to drift from 0.1 we may need to be thoughtful on how to split up the extension script for 0.1 and 1.0 as I imagine 1.0 usage attributes may change and we'll want that tag usage to rely on 1.0 extension and not 0.1 extension.

**Comment by newmuis** [comment]
Resolved the validation merge conflicts.  Any other feedback @honeybadgerdontcare or @choumx?

File: examples/amp-story/progress-bar.html [hunk 2]
```diff
@@ -5,8 +5,7 @@
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <script async custom-element="amp-video" src="https://cdn.ampproject.org/v0/amp-video-0.1.js"></script>
     <script async custom-element="amp-audio" src="https://cdn.ampproject.org/v0/amp-audio-0.1.js"></script>
-    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-0.1.js"></script>
-    <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
```
Comment (honeybadgerdontcare) [inline examples/amp-story/progress-bar.html]: is it intentional to remove amp-analytics here too?

Comment (newmuis) [reply examples/amp-story/progress-bar.html]: Yes, amp-analytics is unused in this example.

**Pull request merged**